src/compiler/watch.ts

```diff
@@ -183,6 +183,7 @@ namespace ts {
         }
 
         let host: DirectoryStructureHost = system;
+        host; // tslint:disable-line no-unused-expression (TODO: `host` is unused!)

```

Comment:
Can you please open a bug to track this so we can look into this. Thanks

Reply:
#26417